kqueen_ui/utils/filters.py

```diff
@@ -26,15 +26,21 @@
 }
 
 
-def cluster_status_icon(status):
-    icon = '<i class="mdi {}" title="{}"></i>'.format(
-        CLUSTER_STATE_MAP.get(status, 'mdi-alert-circle-outline'),
-        status
+def cluster_status_icon(cluster):
+    state = cluster['state']
+    title = cluster['metadata'].get('status_message')
+    if title:
```

Comment:
I suggest to make this ine-line: `title = title.replace('"', "'") if title else state`